source/_integrations/ntfy.markdown

```diff
@@ -97,6 +97,54 @@ data:
 
 {% enddetails %}
 
+## Actions
+
+### Publish notification
+
+For more customizable notifications, you can use the `ntfy.publish` action instead of `notify.send_message`. With `ntfy.publish`, you can take full advantage of the **ntfy** service's capabilities — including setting a priority, adding links, attachments, tags and emoji, and more.
+
+| Data attribute | Optional | Description                                                                                                                                                                                              |
```

Comment:
since all attributes are optional: could the Optional column be removed and this table be converted into a list? And add a note instead stating that all attributes are optional.